src/Setup/Condition/PHPExtensionLoadedCondition.php

```diff
@@ -15,7 +15,7 @@ public function __construct($which)
             function (Setup\Environment $env) use ($which) : bool {
                 return in_array($which, get_loaded_extensions());
             },
-            "ILIAS 6 requires the PHP extension $which."
+            "ILIAS 7 requires the PHP extension $which."
```

Comment:
@klees Is it an option to use `ILIAS_VERSION_NUMERIC` here?

Reply:
Hi @mjansenDatabay!

Yes, it is. I just didn't know of that. Out of curiosity: Where is it defined and who bumps it?

Thanks!

Reply:
@klees I created a PR here https://github.com/ILIAS-eLearning/ILIAS/pull/3204 , where I answered your questions as well.